Source/WebKit/UIProcess/API/glib/WebKitSecurityOrigin.cpp

```diff
@@ -223,8 +231,10 @@ guint16 webkit_security_origin_get_port(WebKitSecurityOrigin* origin)
  * webkit_security_origin_is_opaque:
  * @origin: a #WebKitSecurityOrigin
  *
+ * This function returns %FALSE.  
```

Comment:
Two more unneeded trailing spaces here:

```suggestion
 * This function returns %FALSE.
```